packages/safe-apps-wagmi/package.json

```diff
@@ -22,15 +22,15 @@
   "author": "Safe (https://gnosis-safe.io)",
   "license": "MIT",
   "dependencies": {
-    "@gnosis.pm/safe-apps-provider": "^0.15.0",
-    "@gnosis.pm/safe-apps-sdk": "^7.7.0"
+    "@gnosis.pm/safe-apps-provider": "^0.15.1",
+    "@gnosis.pm/safe-apps-sdk": "^7.8.0"
   },
   "peerDependencies": {
-    "@wagmi/core": ">= 0.3.8 < 0.8.0"
+    "@wagmi/core": "^0.8.18"
```

Comment:
does it really require `^0.8.18` or it can work with any `0.8.x` version? If it can it's better to make the version requirement less restrictive

Reply:
Could you please also update the example wagmi app to use `0.8.x`?